### ISSUE TITLE
Support the latest version of FCL (v0.6.1)?

### ISSUE THREAD

**Issue by yuki-koyama** [BerkeleyAutomation/python-fcl#19]
Hi, thank you for this excellent project. 

I'm wondering if there is a plan to support FCL v0.6.X. The current README says

> Currently, this package is targeted for FCL 0.5.0.

but the latest FCL version available in Homebrew ( https://github.com/Homebrew/homebrew-core/blob/master/Formula/fcl.rb ) is v0.6.1. 

This means that macOS users cannot simply install `python-fcl` through brew and pip:

```bash
brew install fcl # This installs FCL v0.6.1
pip install python-fcl # This generates errors because of the version mismatch!
```

but need to install FCL 0.5.0 manually (e.g., by writing a custom brew file to install FCL v0.5.0).

**Comment by Aniwax** [comment]
Hi @yuki-koyama! 
I am facing the same problem and I build the fcl 0.5.0 package with camke. But do you know how can I make it sync/work together with brew dependencies like libccd, etc? Or in other words, make it a brew package? Can you point me to a resource?

Thanks a lot!

**Comment by yuki-koyama** [comment]
Hi, @Aniwax 

In my case, I didn't build the fcl 0.5.0 by manually running cmake, but created a custom brewfile (`fcl.rb`) based on [this old brewfile](https://github.com/Homebrew/homebrew-core/blob/75dc086c11069896bb2eaebe32bf62d677189ca7/Formula/fcl.rb) and installed fcl via brew:
```
HOMEBREW_NO_INSTALL_CLEANUP=1 brew install fcl.rb
brew switch fcl 0.5.0_1
pip install numpy python-fcl
```
This worked for me, and hope this helps.

**Comment by yuki-koyama** [comment]
And I hope some of these PRs [ #20, #21 ] will resolve the issue soon...

**Comment by Aniwax** [comment]
> Hi, @Aniwax
> 
> In my case, I didn't build the fcl 0.5.0 by manually running cmake, but created a custom brewfile (`fcl.rb`) based on [this old brewfile](https://github.com/Homebrew/homebrew-core/blob/75dc086c11069896bb2eaebe32bf62d677189ca7/Formula/fcl.rb) and installed fcl via brew:
> 
> ```
> HOMEBREW_NO_INSTALL_CLEANUP=1 brew install fcl.rb
> brew switch fcl 0.5.0_1
> pip install numpy python-fcl
> ```
> 
> This worked for me, and hope this helps.

Thank you for the answer! 2 more hints would be great: 
1) The fcl.rb file you created is exactly this file in the link or are there some changes to be adapted? 
2)  Where should I place this fcl.rb file for brew to find it and install it?

Sorry I am not so experienced on installing older packages with brew.

**Comment by Amar1729** [comment]
For visibility - if you prefer using homebrew's fcl formula rather than writing/creating a new one: https://github.com/BerkeleyAutomation/python-fcl/issues/15#issuecomment-735961109

**Comment by manuel-koch** [comment]
I tried to follow the suggested usage of old [fcl.rb](https://github.com/BerkeleyAutomation/python-fcl/issues/19#issuecomment-734835320).

First attempt required to run `brew style --fix fcl.rb` because of

```
Error: fcl: Calling `sha256 "digest" => :tag` in a bottle block is disabled! Use `brew style --fix` on the formula to update the style or use `sha256 tag: "digest"` instead.
```

and afterwards brew complained with

```
$ HOMEBREW_NO_INSTALL_CLEANUP=1 brew install fcl.rb
Error: Failed to load cask: fcl.rb
Cask 'fcl' is unreadable: wrong constant name #<Class:0x00007ff37f869cd0>
Warning: Treating fcl.rb as a formula.
==> Downloading https://ghcr.io/v2/homebrew/core/fcl/manifests/0.5.0_1
#=#=#                                                                         
curl: (22) The requested URL returned error: 404 
Error: Failed to download resource "fcl_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/fcl/manifests/0.5.0_1
```

I'm not familiar with tweaking brew formulas - any hint appreciated !

**Comment by mjd3** [comment]
This should be addressed now with #29; closing but reopen if needed!
